src/app/core/messages/dismissed-message.model.js

```diff
@@ -15,21 +14,24 @@ const mongoose = require('mongoose'),
 /**
  * @type {mongoose.Schema<DismissedMessageDocument, DismissedMessageModel>}
  */
-const DismissedMessageSchema = new mongoose.Schema({
-	messageId: {
-		type: mongoose.Schema.Types.ObjectId,
-		ref: 'Message'
+const DismissedMessageSchema = new mongoose.Schema(
+	{
+		messageId: {
+			type: mongoose.Schema.Types.ObjectId,
+			ref: 'Message'
+		},
+		userId: {
+			type: mongoose.Schema.Types.ObjectId,
+			ref: 'User'
+		}
 	},
-	userId: {
-		type: mongoose.Schema.Types.ObjectId,
-		ref: 'User'
-	},
-	created: {
-		type: Date,
-		default: () => Date.now(),
-		get: util.dateParse
+	{
+		timestamps: {
+			createdAt: 'created',
+			updatedAt: 'updated'
+		}
```

Comment:
Schema didn't have an `updated` field since they are never updated.  The updated timestamp can be disabled by settings `updatedAt` to false.